### DISCUSSION TITLE
Why does it take so long to delete phisher?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#142511] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The following user is a phisher:
https://github.com/kato661/

I reported them already 1.5 weeks ago and the phishing site is still up. I just got an automated mail that it will take some time.

Because the Telegram API key is visible in his code, I have send them thausends of not real randomized email and passwords with the help of a script. But they already blocked a me. Even when changing the IP, it doesn't work anymore.

I also could read that they already phished more than 500 emails.

This email provider "GMX" is currently in the process to change its login system, therefore the site might look legit for a lot of people.

Is there a chance to make a more urgent requests.

**Comment by thomassiard** [comment]
If you've already reported the account and site as phishing through GitHub and are concerned about the response time, you could take additional steps:

Contact GitHub Support Directly:

Since you've already submitted a report, try following up directly with GitHub Support to emphasize the urgency. Use their [contact form](https://support.github.com/contact/report-abuse) and select "Report Abuse" to provide any new, relevant information.
Reach Out to the Hosting Provider:

If the phishing content is hosted outside of GitHub (e.g., on a linked website), reporting the phishing site to the hosting provider might speed up the removal process. You can usually find the hosting provider by performing a WHOIS lookup on the domain.
Report to Other Relevant Entities:

Consider reporting this activity to anti-phishing databases like Google Safe Browsing or APWG (Anti-Phishing Working Group), which helps flag malicious content across multiple platforms.
Taking these additional actions might help expedite the process while GitHub handles your case. Hope this helps!

**Reply by billythekid** [reply]
Are these AI replies?

**Reply by thomassiard** [reply]
No, sir.
